pkg/util/interceptor.go

```diff
@@ -39,3 +44,37 @@ func (s timeoutInterceptor) WrapStreamingHandler(next connect.StreamingHandlerFu
 		return next(ctx, shc)
 	})
 }
+
+// LogRequest logs the request parameters.
+// It logs all kinds of requests.
+func NewLogInterceptor(logger log.Logger) connect.UnaryInterceptorFunc {
+	interceptor := func(next connect.UnaryFunc) connect.UnaryFunc {
+		return connect.UnaryFunc(func(
+			ctx context.Context,
+			req connect.AnyRequest,
+		) (connect.AnyResponse, error) {
+			begin := time.Now()
+			tenantID, err := tenant.ExtractTenantIDFromContext(ctx)
+			if err != nil {
+				tenantID = "unknown"
```

Comment:
I am not sure whether this makes sense. What should I put here if I can not find it?

Reply:
anonymous ?

Reply:
Updated! Thanks!